internal/person/repository.go

```diff
@@ -125,3 +129,78 @@ func (repo *personRepository) CreateProfile(ctx context.Context, iamID string, p
 
 	return person, nil
 }
+
+func (repo *personRepository) EditProfile(ctx context.Context, iamID string, params ProfileParams) (*models.Person, error) {
+	u1, err := uuid.NewV4()
+	if err != nil {
+		return nil, err
+	}
+
+	u2, err := uuid.NewV4()
+	if err != nil {
+		return nil, err
+	}
+
+	person, err := models.People(
+		qm.Load(models.PersonRels.PersonProfiles),
+		qm.Load(models.PersonRels.JobProvider),
+		models.PersonWhere.IamID.EQ(iamID)).One(ctx, repo.executor)
+
+	if err != nil {
+		return nil, err
+	}
+
+	var profileID = u1.String()
+
+	if len(person.R.PersonProfiles) > 0 {
+		profileID = person.R.PersonProfiles[0].ID
+	}
+
+	personProfile := &models.PersonProfile{
+		ID:          profileID,
+		PersonID:    person.ID,
+		ProfileURL:  "dfds", // TODO: This is to be changed later.
+		Headline:    null.StringFrom(params.Headline),
```

Comment:
Should it be made possible to change the following fields `first_name`, `last_name`, `avatar_url`, `email` as well?

Reply:
@hanmd82 I rebased and force-pushed the code. Check the latest commits. I have modify the code to allow editing the above fields. However, I think we should disable editing the `email` otherwise need to have an email verification flow.